src/Centreon/Domain/PlatformInformation/Model/PlatformInformation.php

```diff
@@ -100,6 +105,24 @@ public function setRemote(bool $isRemote): self
         return $this;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getPlatformName(): ?string
+    {
+        return $this->platformName;
+    }
+
+    /**
+     * @param string|null $name
+     * @return self
+     */
+    public function setPlatformName(?string $name): self
+    {
+        $this->platformName = filter_var($name, FILTER_SANITIZE_STRING);
```

Comment:
could throw an exception before converting a central to a remote with an empty name ?
and to avoid to patch with an empty name

Reply:
those checks are handled in the POST.